pdcupdater/handlers/depchain/base.py

```diff
@@ -20,6 +20,16 @@ class BaseKojiDepChainHandler(pdcupdater.handlers.BaseHandler):
     parent_type = None
     child_type = None
 
+    def construct_topic(self, config):
+        # Return a single hardcoded topic when using STOMP
+        if config.get('stomp_uri'):
```

Comment:
Hm, yes.  If both stomp_uri is present and zmq is enabled, we should raise an Exception of some kind since that isn't a sensible scenario for pdc-updater.